pkg/components/state/pluggable.go

```diff
@@ -321,6 +322,18 @@ func (ss *grpcStateStore) Multi(ctx context.Context, request *state.Transactiona
 	return err
 }
 
+// MultiMaxSize returns the maximum number of operations allowed in a transactional request.
+func (ss *grpcStateStore) MultiMaxSize() int {
+	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
```

Comment:
Value should be cached, as it should be a constant

Reply:
Done